DigitalLearningSolutions.Web/Attributes/NoStoreAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DigitalLearningSolutions.Web.Attributes
+{
+    using Microsoft.AspNetCore.Mvc.Filters;
+
+    public class NoStoreAttribute : ActionFilterAttribute

```

Comment:
I'd be inclined to call it something like `NoCaching` instead of `NoStore` for readability - `NoStore` is a bit more vague and developers might not work out what it's doing when they see it on an endpoint so quickly.
I get why you haven't gone for `NoCache`, but I think `NoCaching` or `NotCacheable` are sufficiently different from the "no-cache" directive not to cause confusion.